proto/viam/module/v1/module.proto

```diff
@@ -54,3 +58,11 @@ message ReadyResponse {
   bool ready = 1;
   HandlerMap handlermap = 2;
 }
+
+message ValidateRequest {
+  viam.app.v1.RobotConfig config = 1;
```

Comment:
why RobotConfig?

Reply:
We should use `ComponentConfig`. Updated.